app/controllers/reviewable_claimed_topics_controller.rb

```diff
@@ -38,7 +38,7 @@ def destroy
   private
 
   def notify_users(topic, claimed_by)
-    user_ids = User.staff.pluck(:id)
+    user_ids = User.human_users.staff.pluck(:id)
 
     if SiteSetting.enable_category_group_moderation? && group_id = topic.category&.reviewable_by_group_id.presence
       user_ids.concat(GroupUser.where(group_id: group_id).pluck(:user_id))
```

Comment:
Same as https://github.com/discourse/discourse/pull/19179/files#r1031885962 because we can just restrict the message to a certain group.